docs/class5/connexml.md

```diff
@@ -417,6 +428,62 @@ An existing call is transferred to a different destination using the `Dial` ver
             </Dial>
         </Response>
         ```
+    
+    10. **Voicemail**
+        ``` xml
+        <?xml version="1.0" encoding="UTF-8"?>
+        <Response>
+            <Dial>
+                <Voicemail />
+            </Dial>
+        </Response>
+        ```
+    
+    11. **Voicemail Inbox**
+        ``` xml
+        <?xml version="1.0" encoding="UTF-8"?>
+        <Response>
+            <Dial>
+                <Voicemail inbox="true">
+                 <Voicemail/>
+            </Dial>
+        </Response>
+        ```
+
+    12. **Barge (Default Method)**
+        ``` xml
+        <?xml version="1.0" encoding="UTF-8"?>
+        <Response>
+            <Dial>
+                <Barge></Barge>
+            </Dial>
+        </Response>
+        ```
+    
+    13.  **Barge**
+        ``` xml
+        <?xml version="1.0" encoding="UTF-8"?>
+        <Response>
+            <Dial>
+                <Barge>
+                    whisper="a" // enables whisper mode in aleg
+                    bridge="b" // enables listen to in bleg
+                    commands= "false" // false means no DTMF signals/commands during eavesdrop
+                </Barge>
+            </Dial>
+        </Response>
+        ```
+
+    14. **Barge (Group)** 
+        ``` xml
+        <?xml version="1.0" encoding="UTF-8"?>
+        <Response>
+            <Dial>
+                <Barge
+                >GROUPA</Barge> // Add a group of customers in SIP Profile to listen to the conversation of that particular group. If no group is mentioned then you can listen to any ongoing calls 
+            </Dial>
+        </Response>
+        ```
```

Comment:
<?xml version="1.0" encoding="UTF-8"?>
<Response>
    <Dial>
        <Barge>GROUPA</Barge> <!-- Listens to the conversation of the specified group. If no group is mentioned, it can listen to any ongoing calls for this customer. -->
    </Dial>
</Response>

Reply:
done